app/services/foreman_templates/template_importer.rb

```diff
@@ -6,15 +6,12 @@ class TemplateImporter < Action
     attr_accessor :metadata, :name, :text
 
     def self.setting_overrides
-      super + %i(associate)
+      super + %i(associate force)
     end
 
     def initialize(args = {})
       super
-      # Rake hands off strings, not booleans, and "false" is true...
-      if @verbose.is_a?(String)
-        @verbose = @verbose != 'false'
-      end
+      parse_bools('@verbose', '@force')
```

Comment:
I'm happy you extracted the logic to method but I'd prefer if you remained explicit about variables and avoid using `instance_variable_set` as it's hard to read, how about

```ruby
@verbose = parse_bool(@verbose)
@force  = parse_bool(@force)
```